.claude/plans/2026-03-17-Plan1-NotifyAndSay.md

```
@@ -0,0 +1,238 @@
+git commit -m "feat: add tools say CLI with volume and mute support"
+```
+
+### Task 6: Wire `sayy` alias in zsh install
+
+**Files:**
+- Modify: `src/zsh/features/port.ts` (or create new feature, or add to install flow)
+
+**Step 1: Add sayy alias**
+
+In the zsh hook or as a new feature, add:
+```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)